src_python/models/supervised.py

```diff
@@ -153,10 +158,9 @@ def evaluate(self):
     args = dict(zip(arg_names, sys.argv[1:]))
 
     nodes = pd.read_csv(args["path_nodes"],index_col=0)
-    #nodes = nodes.astype("float32")
-
     edges = pd.read_csv(args["path_edges"])
-    #edges = edges.astype({"source":"uint32","target":"uint32"})
+    edges = edges.astype({"source":"uint32","target":"uint32"})
```

Comment:
Add space in between the comma of `"source":"uint32","target":"uint32"`

Reply:
Updated in 577a1b78625e555cc93d60b9e1c228723d8e0fd5